### ISSUE TITLE
Images in events causing layout issues on "load more"

### ISSUE THREAD

**Issue by davidpagepsycle** [stephenharris/Event-Organiser-Posterboard#3]
Although $container.imagesLoaded is used when the poster board events are first loaded, calls to eventorganiser_fetch_events() doesn't wait for images to load. Our fix was to wrap https://github.com/stephenharris/Event-Organiser-Posterboard/blob/master/js/event-board.js#L239 as 
```php
			$container.imagesLoaded( function() {
				$container.masonry( 'reload' );
			});
```

**Comment by stephenharris** [comment]
Thanks @davidpagepsycle will implement this fix shortly :).

**Comment by stephenharris** [comment]
Fixed with 151091185725cd74bf3afe387dc9e2bd0e899654
